barrels/README.md

```diff
@@ -1,8 +1,25 @@
 # Barrels
-A collection of Connect IQ libraries.
+A collection of Connect IQ libraries. For more details on what Monkey Barrels are and how to define
+them checkout the [Monkey Barrels](https://developer.garmin.com/connect-iq/programmers-guide/monkey-barrels)
+section of the Programmer's Guide.
 
-## Semicircles
-This barrel provides an abstract coordinate type that internally uses a semicircles representation.
+## Including Monkey Barrels
+To include a Monkey Barrel in your project you need to declare a dependency on that Barrel within
+your Manifest file and then link to the Barrel within your project's Jungle file. See the
+[Programmer's Guide](https://developer.garmin.com/connect-iq/programmers-guide/monkey-barrels#how-to-include-barrels)
+for a detailed description of how to include Monkey Barrels in your project.
+
+## Barrel Index
+
+### LogMonkey
+The [LogMonkey barrel](https://github.com/garmin/connectiq-apps/tree/master/barrels/LogMonkey)
+provides some basic logging utilities to aid in developing Connect IQ apps. Inclusions of the
+LogMonkey within an app will automatically be compiled out or minimized when building a release
```

Comment:
Compiled out or minimized?

Reply:
I'm going to change the entire sentence to be: Including LogMonkey within an app and building the app in release mode will automatically minimize the compiled output to save run time memory.